indexd/drs/blueprint.py

```diff
@@ -248,17 +248,19 @@ def bundle_to_drs(record, expand=False, is_content=False):
 
 def parse_checksums(record, drs_object):
     ret_checksum = []
-    if "hashes" in record:
-        for k in record["hashes"]:
-            ret_checksum.append({"checksum": record["hashes"][k], "type": k})
-    elif "checksum" in record:
-        checksums = json.loads(record["checksum"])
-        for checksum in checksums:
-            ret_checksum.append(
-                {"checksum": checksum["checksum"], "type": checksum["type"]}
-            )
-
-    return ret_checksum
+    try:
+        if "hashes" in record:
+            for k in record["hashes"]:
+                ret_checksum.append({"checksum": record["hashes"][k], "type": k})
+        elif "checksum" in record:
```

Comment:
shouldn't it be "checksumS"?

Reply:
the column name is checksum :upside_down_face: